src/cimplog-rdk-logger.c

```diff
@@ -43,11 +53,13 @@ void __cimplog(const char *module, int level, const char *msg, ...)
 			buf[nbytes] = '\0';
 		}
 		
-
 		RDK_LOG(_level[0x3 & level], rdk_logger_module, "%s: %s", module, buf);
     }
+
+    return;
 }
 
+// The below 'weak' linkage should be replaced with strong definition in the module integrating to cimplog
```

Comment:
This comment should have been addressed in the .h file. If not already so, should be added in the .h file.